lib/cli/src/Cardano/CLI.hs

```diff
@@ -375,9 +379,9 @@ cmdWalletCreate = command "create" $ info (helper <*> cmd) $ mempty
             getLine prompt parser
         wSndFactor <- do
             let prompt =
-                    "(Enter a blank line if you do not wish to use a second \
-                    \factor.)\n\
-                    \Please enter a 9–12 word mnemonic second factor: "
+                    "(Enter a blank line if you do not wish to use a second " <>
+                    "factor.)\n" <>
+                    "Please enter a 9–12 word mnemonic second factor: "
```

Comment:
Just out of curiosity, what's the reason for changing the line continuation style here?

Reply:
The `CPP` language extension.

Reply:
> The `CPP` language extension.

Understood! At least our use of the CCP extension is limited to just a single file.